quizgen/pdf.py

```diff
@@ -165,5 +170,5 @@ def set_cli_args(parser):
     parser.add_argument('--seed', dest = 'seed',
         action = 'store', type = int, default = None,
         help = 'The random seed to use (defaults to a random seed).')
-
-    return parser
+    
+    return parser
```

Comment:
Missing a final newline.
I would check your editor.

Note that I am not saying we want a blank line at the end of every file,
but every line should end with a line terminator (and you are missing that).